packages/boxel-ui/addon/src/components/filter-list/index.gts

```diff
@@ -0,0 +1,63 @@
+import { fn } from '@ember/helper';
+import { on } from '@ember/modifier';
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+
+import { cn, eq } from '../../helpers.ts';
+
+export type Filter = {
+  displayName: string;
+  query: any;
```

Comment:
Since `FilterList` never actually uses `query`, does it need to have an opinion about anything in `Filter` other than `displayName`? This could be an interface that leaves everything else up to the consumer.